### PR TITLE
Update testing-guide.md

### PR DESCRIPTION
**Pull request by alknopfler**
Add description for command line using virt-install

### CODE REVIEW

File: docs/testing-guide.md [hunk 1]
```diff
@@ -110,6 +110,20 @@ Unlike the installer ISO, there isn't anything specific that needs to be done wh
...
 
+## Testing using virt-install (CLI)
+
+If you want to use a command line example to deploy the VM and test it with your EIB image generated, you could use the following commands:
+
+- First, we need to create a disk empty to be used by the VM:
+
+`qemu-img create -f qcow2 example.img 6G`
+
+- Then, we could use virt-install to create and define a VM using the EIB output image generated:
```
Comment (jdob) [inline docs/testing-guide.md:121]: Following on the previous comment: "Next, use `virt-install` to define and run a VM using the EIB generated image:"

File: docs/testing-guide.md [hunk 2]
```diff
@@ -110,6 +110,20 @@ Unlike the installer ISO, there isn't anything specific that needs to be done wh
...
+
+`virt-install --name testVM --memory 4096 --vcpus 4 --disk ./example.img --install no_install=yes --cdrom ./eib-image-generated.iso --network default --osinfo detect=on,name=sle-unknown`
+
+After executing this command, the first time, you will need to install the OS following the instructions. After installing the OS a reboot will happen. The second time booting the system, you will need to choose the `Boot From Disk` option.
```
Comment (jdob) [inline docs/testing-guide.md:125]: Still following the suggestions around the tone of the doc: "During the first boot, you'll need to install the OS following the instructions. After installation, a reboot will happen. Subsequent boots will use the "Boot From Disk" option to boot from the installed OS."

File: docs/testing-guide.md [hunk 3]
```diff
@@ -110,6 +110,20 @@ Unlike the installer ISO, there isn't anything specific that needs to be done wh
...
+
+If you want to use a command line example to deploy the VM and test it with your EIB image generated, you could use the following commands:
+
+- First, we need to create a disk empty to be used by the VM:
+
+`qemu-img create -f qcow2 example.img 6G`
+
+- Then, we could use virt-install to create and define a VM using the EIB output image generated:
+
+`virt-install --name testVM --memory 4096 --vcpus 4 --disk ./example.img --install no_install=yes --cdrom ./eib-image-generated.iso --network default --osinfo detect=on,name=sle-unknown`
```
Comment (jdob) [inline docs/testing-guide.md:123]: Can you explicitly add line breaks to make this more consistently readable? I'm thinking of something like:

```
virt-install \
--name testVM \
--memory 4096 \
--vcpus 4 \
--disk ./example.img \
--install no_install=yes \
--cdrom ./eib-image-generated.iso \
--network default \
--osinfo detect=on,name=sle-unknown`
```

**Review by jdob — COMMENTED** [review]
Thanks for adding this. A few suggestions to try to keep the tone consistent throughout the guide.

File: docs/testing-guide.md [hunk 4]
```diff
@@ -110,6 +110,20 @@ Unlike the installer ISO, there isn't anything specific that needs to be done wh
 a raw image. EIB can be used to overwrite the raw image attached to a VM and, when the VM is booted, it will use
 the newly built image.
 
+## Testing using virt-install (CLI)
```
[the remark this answers is not in the document]
Comment (jdob) [reply docs/testing-guide.md:113]: Most are suggestions, but this is the only one I'd consider blocking the PR from landing.

[the remark this answers is not in the document]
Comment (alknopfler) [reply docs/testing-guide.md:113]: done!

File: docs/testing-guide.md [hunk 5]
```diff
@@ -110,6 +110,20 @@ Unlike the installer ISO, there isn't anything specific that needs to be done wh
...
+
+If you want to use a command line example to deploy the VM and test it with your EIB image generated, you could use the following commands:
```
[the remark this answers is not in the document]
Comment (alknopfler) [reply docs/testing-guide.md:115]: thanks!

File: docs/testing-guide.md (hunk 3 above)
Comment (alknopfler) [reply docs/testing-guide.md:123]: you're right

File: docs/testing-guide.md (hunk 2 above)
Comment (alknopfler) [reply docs/testing-guide.md:125]: changed

**Review by jdob — APPROVED** [review]

**Pull request merged**